tests/test_browser_transient_dismantle_lifecycle_regression.py

```
@@ -0,0 +1,82 @@
+2) BrowserPanel.close() performs permanent detach from BrowserWindowPortalRegistry.
+"""
+
+from __future__ import annotations
+
+import subprocess
+from pathlib import Path
+
+
+def repo_root() -> Path:
+    result = subprocess.run(
+        ["git", "rev-parse", "--show-toplevel"],
```

Ruff (0.15.2)

[error] Starting a process with a partial executable path

(S607)